cfgov/jinja2/v1/events/_macros.html

```diff
@@ -155,16 +155,6 @@ <h2 class="event-venue_heading">
                 {% else %}
                     {% import 'macros/time.html' as time %}
                     {{ time.render(event.start_dt) }}
-
-                    {# TODO: Replace with real download link #}
```

Comment:
If the ICS is ever coming back, it might be worth commenting out the code rather than deleting it.